graphqlapi/src/main/java/io/stargate/graphql/schema/fetchers/dml/DmlFetcher.java

```diff
@@ -96,34 +97,36 @@ protected DmlFetcher(
       Column column = getColumn(table, clauseEntry.getKey());
       for (Map.Entry<String, Object> condition : clauseEntry.getValue().entrySet()) {
 
+        ColumnRelationBuilder<Relation> relationStart = Relation.column(column.name());
+        Relation relation;
         if (condition.getKey().equals("in")) {
-          relations.add(
-              Relation.column(column.name()).in(buildListLiterals(column, condition.getValue())));
-        }
```

Comment:
Missing return statement. It made IN relations fail for UDTs, and it was probably also the case for scalar types 😞